lib/tasks/data_explorer.rake

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+require 'rainbow'
```

Comment:
This is a problem rainbow is a dependency of rubocop, it may go away and it is only there in dev and test environments. 

Recommend you remove it for now, in general we don't have any rake tasks that colorize and colorizing has its downsides (you don't want to get weird chars in text logs)

Reply:
Will do, thank you